src/wallet/monero_wallet_full.h

```diff
@@ -252,11 +252,15 @@ namespace monero {
     std::string get_keys_file_buffer(const epee::wipeable_string& password, bool view_only) const;
     std::string get_cache_file_buffer() const;
 
+  // --------------------------------- PROTECTED --------------------------------
+
+  protected:
+      std::unique_ptr<tools::wallet2> m_w2;            // internal wallet implementation
```

Comment:
Please shift two spaces to the left to be aligned with surrounding variables.

Reply:
Done!